qiita_db/metadata_template/util.py

```diff
@@ -113,7 +113,7 @@ def prefix_sample_names_with_id(md_template, study_id):
         # Create a new column on the metadata template that includes the
         # metadata template indexes prefixed with the study id
         md_template['sample_name_with_id'] = (study_ids + '.' +
-                                              md_template.index)
+                                              np.array(md_template.index))
```

Comment:
`md_template.index.values` should just return the underlying numpy array, without having to create a copy.